src/libs/migrateOnyx.js

```diff
@@ -0,0 +1,31 @@
+import RenameActiveClientsKey from './migrations/RenameActiveClientsKey';
+
+export default function () {
+    const startTime = Date.now();
+    console.debug('[Migrate Onyx] start');
+
+    return new Promise((resolve) => {
+        // Add all migrations to an array so they are executed in order
+        const migrationPromises = [
+            RenameActiveClientsKey,
+        ];
+
+        // Reduce all promises down to a single promise. All promises run in a linear fashion, waiting for the
+        // previous promise to finish before moving onto the next one.
+        migrationPromises
+            /* eslint-disable arrow-body-style */
+            .reduce((previousPromise, migrationPromise) => {
+                return previousPromise.then(() => {
+                    return migrationPromise();
+                });
+            }, Promise.resolve())
+            /* eslint-enable arrow-body-style */
```

Comment:
Why are we doing this? Could we change the setting of this linter? Maybe to `as-needed`? (I am unsure how to check the current setting :ohnothing:)

Reply:
Well, the `eslint-disable` is the mechanism we use for "as-needed". If you end up making this code pass the linter, it is nearly impossible to tell what it's doing, it's super ugly, so in cases where you have multiple nested arrow functions, each with an implied return, I think it's better to turn off the linter and make the code more readable. It's a pretty rare situation though and this lint rule is good for 98% of use cases.

Reply:
Got it